src/components/MenuPanel/MenuPanel.js

```diff
@@ -105,8 +134,47 @@ class MenuPanel extends React.PureComponent {
                     : this.renderAppGroup(app, false)
                 )}
               </div>
-              <h1 style={{ marginTop: '24px' }}>System</h1>
-              <div>{systemApps.map(app => this.renderAppGroup(app, true))}</div>
+              <StyledButton onClick={this.handleToggleSystemApps}>
+                <h1
+                  style={{
+                    marginTop: '24px',
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    alignItems: 'flex-end',
+                  }}
+                >
+                  <span>System</span>
+                  <span
+                    css={`
+                      transform: rotate(${systemAppsOpened ? 180 : 0}deg);
+                      position: relative;
+                      top: ${systemAppsOpened ? -5 : 0}px;
+                      font-size: 7px;
+                      opacity: 0.7;
+                    `}
+                  >
+                    <IconArrow />
+                  </span>
+                </h1>
+              </StyledButton>
+              <Transition
```

Comment:
I'm not familiar with react-spring, but why do we pass a boolean and then read from scope rather than pass the actual items to render (systemApps), e.g.
```suggestion
              { systemAppsOpened && <Transition
                items={systemApps.map(app => this.renderAppGroup(app, true))}
```

This is based on what I read in the docs:
> You should always rely on the item that you receive instead of reading from scope, since you could be rendering an item that has already been deleted, but is retained inside Transition until its fade-out is complete. This is especially important for routes.

Reply:
There are two other ways to use [Transition](https://www.react-spring.io/docs/props/transition#/red) and I've just gotten accustomed to using it in this way (passing in a boolean).
I can't really say if there is a con/benefit from either approach. cc/ @bpierre (Pierre is our in.-house `react-spring` expert)

Reply:
This is exactly why `Transition` is needed: it makes it possible to retain the rendered component until the leaving transition has completed. If `Transition` only gets rendered if `systemAppsOpened` is `true`, there would only be an entering transition.